internal/controller/storageclient_controller.go

```diff
@@ -212,6 +212,11 @@ func (r *StorageClientReconciler) reconcilePhases() (ctrl.Result, error) {
 		return res, err
 	}
 
+	if r.storageClient.Status.ConsumerConfig.Hash != storageClientResponse.DesiredConfigHash {
+		r.storageClient.Status.ConsumerConfig.Hash = storageClientResponse.DesiredConfigHash
+		r.storageClient.Status.ConsumerConfig.Timestamp = metav1.Now()
+	}
```

Comment:
shouldn't this preferably be at the end of reconcile? In current logic and updated hash isn't suggesting a successful reconcile, isn't it?

Reply:
I can move it to end not a problem

Reply:
Done.